tasking/pulp/tasking/services/scheduler.py

```diff
@@ -9,15 +10,20 @@
 from django.utils import timezone
 from django.db import IntegrityError
 
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", 'pulp.app.settings')
+
+# This import is here so that Celery will find our application instance. It's important that other
+# pulp and django code not get used until after the celery app is instantiated and does its "fixup"
+# of django.
```

Comment:
I do not think that order of the imports is important in scheduler.
To start celerybeat apart form scheduler you specify `--app=pulp.tasking.celery_app:celery` and it will be handled first. As for any other workers the first thing celery will do is import this app module and start worker. All the fixups will happen then. So at the time of scheduler import, everything will be ready and imports can go here in any order. If it was not the case you would not able to import models outside of callables here. That's my understanding.